x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/common/if.ts

```diff
@@ -20,9 +20,10 @@ export function ifFn(): ExpressionFunction<'if', any, Arguments, any> {
     help,
     args: {
       condition: {
-        types: ['boolean', 'null'],
```

Comment:
Yes because I don't think it makes sense to pass `null` as a `condition` to the `if` function, so we shouldn't encourage it by listing it as a valid data  type for the `condition` arg in the docs. Even with this change, the interpreter will cast `null` to `false` so this isn't a breaking change, and the functionality of an expression like `if null then="foo" else="bar"` won't change.

Reply:
>  Even with this change, the interpreter will cast null to false so this isn't a breaking change

Ah, of course. Thanks for clarifying!